ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -173,10 +192,51 @@
   [context]
   (bulk-update/cleanup-old-bulk-update-status context))
 
+(defn- process-subscriptions
+  "Process each subscription in subscriptions."
+  [context subscriptions time-constraint]
+  (for [subscription subscriptions
+       :let [email-address (:email-address subscription)
+             coll-id (:collection-concept-id subscription)
+             query-params (create-query-params (:metadata subscription))
+             params1 (merge {"created-at" time-constraint}
+                            {"collection-concept-id" coll-id}
+                            query-params)
+             params2 (merge {"revision-date" time-constraint}
+                            {"collection-concept-id" coll-id}
+                            query-params)
+             gran-ref1 (search/find-granule-references context params1)
+             gran-ref2 (search/find-granule-references context params2)
```

Comment:
Assuming that these searches will be tested in later PRs.

Reply:
find-granule-references is well tested existing function/end point.  If your parameter is correct, then you should expect the function to work.  I did the unit test to cover the parsing of the parameters.

Reply:
plus, once we decide what needs to be included in the email, we might need to call a different end point...